scripts/deploy.sh

```diff
@@ -85,6 +85,10 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
     sed -i "s/aio1/$(hostname)/" /etc/openstack_deploy/openstack_user_config.yml
     sed -i "s/aio1/$(hostname)/" /etc/openstack_deploy/conf.d/*.yml
   fi
+  if [ "$DEPLOY_CEILOMETER" != "yes" ]; then
+    rm -f /etc/openstack_deploy/conf.d/ceilometer.yml
+    sed -i 's/swift_ceilometer_enabled:.*/swift_ceilometer_enabled: false/' /etc/openstack_deploy/user_variables.yml
```

Comment:
The bootstrap-host role overrides the OSA default so it is always set, it this wasn't happening we wouldn't need to override the value.